pkgs/development/python-modules/dill/default.nix

```diff
@@ -1,48 +1,38 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, pytestCheckHook
+, python
+, setuptools
 }:
 
 buildPythonPackage rec {
```

Comment:
```suggestion
python.pkgs.buildPythonPackage rec {
```

Reply:
Thanks for reviewing! Is this a new pattern we're looking to change to? I haven't seen it before.